src/locale/cd-lang-picker.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+  <div class="row">
+    <div class="col-md-4 col-sm-6">
+      <span class="fa fa-language fa-2x cd-lang-picker__icon"></span>
+      <select class="cd-lang-picker__select" v-model="lang">
+        <option v-for="availableLanguage in availableLanguages" :value="availableLanguage.value">{{ availableLanguage.name }}</option>
+      </select>
+    </div>
+    <div class="col-md-8 col-sm-6">
```

Comment:
This image/"help us translate"  could have been left in the template/footer rather than in this component: it has no logic nor is related to picking a language